tests/_support/Log/Handlers/TestHandler.php

```diff
@@ -35,8 +37,10 @@ public function __construct(array $config)
     {
         parent::__construct($config);
 
+        Time::setTestNow('2023-11-25 12:00:00');
```

Comment:
Remove this line.

This always sets the date/time `'2023-11-25 12:00:00'` when we use `TestHandler`.
This is superfluous.

If we want to fix the time, we should do so in the test code.

Reply:
Done.